Published version 0.1.0.

christianareas opened the pull request.

christianareas committed:
Updated package-lock.json.

christianareas committed:
Replaced bloated starter with bare-bones starter.

christianareas committed:
Set up bare-bones site.

ghost commented:
# [Gatsby Cloud](https://www.gatsbyjs.com) Build Report
**gatsby-sandox-publish**

:tada: Your build was successful! [See the Deploy preview here.](https://gatsbysandoxpublish-preview.gtsb.io)

## Build Details

[View the build logs here.](https://gatsbyjs.com/dashboard/2fd6dd1d-e5ba-4224-9f78-f665e25eb9d6/sites/a63e1e6f-2420-46b8-9592-b5b4f87e9c51/builds/ddcfa593-337e-4878-9bb1-8f5ecc5a78fe/details)

:clock1: Build time: **50s**

## Performance

### Lighthouse report

| Metric | Score |
| ----- | ---- |
| Performance |   :green_heart: 94   |
| Accessibility |   :green_heart: 100   |
| Best Practices |   :green_heart: 100   |
| SEO |   :large_orange_diamond: 83   |

:link: [View full report](https://build-ddcfa593-337e-4878-9bb1-8f5ecc5a78fe.gtsb.io/reports/lighthouse/index.html)

christianareas committed:
Set up site (kept it simple for 0.1.0).

Fixed build issues by adding the blog.

Simplified home page and turned off animation.

Changed name of blog and removed posts from home page.

Tidied up the homepage.

christianareas committed:
Fixed createPages issue (temporarily).

christianareas committed:
Fixed build issues by adding the blog.

christianareas committed:
Simplified home page and turned off animation.

christianareas committed:
Changed name of blog and removed posts from home page.

christianareas committed:
Tidied up the homepage.

christianareas committed:
Merge branch 'preview' of github.com:christianareas/gatsby-sandox into preview
